pallets/manta-pay/src/lib.rs

```diff
@@ -109,13 +109,16 @@ pub mod runtime;
 #[frame_support::pallet]
 pub mod pallet {
     use super::*;
-    use frame_support::pallet_prelude::*;
+    use frame_support::{pallet_prelude::*, traits::StorageVersion};
     use frame_system::pallet_prelude::*;
     use sp_runtime::traits::AccountIdConversion;
 
+    const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Not sure all pallets have the same storage version as `1`.
You better check it on chain.

Reply:
Our pallets don't use storage_version yet, so I believe there's no any version storage here.

Do you mean all pallets starting with `1` is not a good choice?

Reply:
Agree with Yi - except for `manta_collator_selection` where STORAGE_VERSION is already used.

Reply:
I've check it.
for some history reason, there's already some existing `storage_version` info like:
```
2022-08-05 17:03:33.521 DEBUG                 main runtime: "----PreUpgrade----"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "pallet_asset_manager"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "AssetManager"
2022-08-05 17:03:33.521 DEBUG                 main runtime: StorageVersion(1)
2022-08-05 17:03:33.521 DEBUG                 main runtime: "----PreUpgrade----"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "pallet_tx_pause"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "TransactionPause"
2022-08-05 17:03:33.521 DEBUG                 main runtime: StorageVersion(0)
2022-08-05 17:03:33.521 DEBUG                 main runtime: "----PreUpgrade----"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "manta_collator_selection"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "CollatorSelection"
2022-08-05 17:03:33.521 DEBUG                 main runtime: StorageVersion(1)
2022-08-05 17:03:33.521 DEBUG                 main runtime: "----PreUpgrade----"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "calamari_vesting"
2022-08-05 17:03:33.521 DEBUG                 main runtime: "CalamariVesting"
2022-08-05 17:03:33.522 DEBUG                 main runtime: StorageVersion(0)
```

But It doesn't matter. I will reset all value into 1 one time.